### PR TITLE
Always redirect to widget list after deleting a widget

### PR DESCRIPTION
**Pull request by elenatorro**
Related issue: https://github.com/CartoDB/cartodb/issues/13485

### CODE REVIEW

File: lib/assets/test/spec/builder/editor/widgets/widgets-service.spec.js [hunk 1]
```diff
@@ -33,4 +37,40 @@ describe('editor/widgets/widgets-service', function () {
...
+    });
+
+    describe('when the widgetDefinitionModel is defined', function () {
+      it('should create a delete confirmation widget template', function () {
+        spyOn(WidgetsService._modals, 'create').and.callThrough();
+
+        WidgetsService.removeWidget(widgetModel);
+
+        expect(WidgetsService._modals.create).toHaveBeenCalled();
+        expect($('.js-cancel').text().trim()).toEqual('editor.widgets.delete.cancel');
```
[the remark this answers is not in the document]
Comment (rjimenezda) [reply lib/assets/test/spec/builder/editor/widgets/widgets-service.spec.js]: We get it, hound.

**Comment by ramiroaznar** [comment]
@elenatorro LGTM! :boom: :boom: :boom:

File: lib/assets/javascripts/deep-insights/widgets/histogram/content-view.js [hunk 2]
```diff
@@ -107,7 +107,7 @@ module.exports = CoreView.extend({
 
   _onWindowResized: function () {
     if (window.innerWidth < TABLET_VIEWPORT_BREAKPOINT) {
-      this.histogramChartView.forceResize();
+      this.histogramChartView && this.histogramChartView.forceResize();
```
Comment (rjimenezda) [inline lib/assets/javascripts/deep-insights/widgets/histogram/content-view.js]: great idea 👏

File: lib/assets/javascripts/builder/editor/widgets/widgets-service.js [hunk 3]
```diff
@@ -18,22 +18,24 @@ var service = (function () {
       checkAndBuildOpts(opts, REQUIRED_OPTS, this);
     },
 
-    removeWidget: function (widgetDefModel, callback) {
-      var widgetName = widgetDefModel.get('title');
+    removeWidget: function (widgetDefinitionModel) {
+      if (widgetDefinitionModel) {
+        var self = this;
```
Comment (rjimenezda) [inline lib/assets/javascripts/builder/editor/widgets/widgets-service.js]: I was going to comment on this, then I remembered we only have es6 on dashboard still 🤦‍♂️

Comment (elenatorro) [reply lib/assets/javascripts/builder/editor/widgets/widgets-service.js]: Life is hard without ES6

**Pull request merged**